src/opencl/ssh_kernel.cl

```diff
@@ -0,0 +1,192 @@
+/*
+ * This software is Copyright (c) 2018 Dhiru Kholia, Copyright (c) 2018 magnum,
+ * and it is hereby released to the general public under the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ */
+
+#include "opencl_device_info.h"
+#include "opencl_misc.h"
+#include "opencl_asn1.h"
+#include "opencl_des.h"
+#include "opencl_md5_ctx.h"
+#define AES_SRC_TYPE MAYBE_CONSTANT
+#include "opencl_aes.h"
+
+#ifndef PLAINTEXT_LENGTH
+#error PLAINTEXT_LENGTH must be defined
+#endif
+
+// input
+typedef struct {
+	uint length;
+        uchar v[PLAINTEXT_LENGTH];
+} ssh_password;
```

Comment:
looks like some kind of indentation problem here

Reply:
Yeah, the code is in very rough state at the moment. I will be fixing it soon.